Problem building from source - not able to find org.eclipse.ease.feature.feature.group

### Discussed in https://github.com/debrief/debrief/discussions/5168

<div type='discussions-op-text'>

<sup>Originally posted by **bradh** January 19, 2023</sup>
I'm trying to build from source (using command line maven), but hitting this:

```
[INFO] Scanning for projects...
[INFO] Resolving target definition file:/home/bradh/coding/debrief/org.mwc.debrief.targetplatforms/eclipse-latest.target for environments=[linux/gtk/x86_64, win32/win32/x86_64, macosx/cocoa/x86_64], include source mode=honor, execution environment=StandardEEResolutionHints [executionEnvironment=OSGi profile 'JavaSE-11' { source level: 11, target level: 11}], remote p2 repository options=org.eclipse.tycho.p2.remote.RemoteAgent@5cd71ee2...
[ERROR] Failed to resolve target definition file:/home/bradh/coding/debrief/org.mwc.debrief.targetplatforms/eclipse-latest.target: Could not find "org.eclipse.ease.feature.feature.group/0.8.0.I202105061131" in the repositories of the current location -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException
```

Any ideas for a source for that?</div>

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -25,7 +25,7 @@ jobs:
                 sudo apt-get update -y
                 sudo apt-get install -y libsqlite3-mod-spatialite
                 sudo apt-get install libspatialite-dev
-                sudo apt-get install libpq-dev python-dev
+                sudo apt-get install libpq-dev python2-dev
                 sudo add-apt-repository -y ppa:ubuntugis/ubuntugis-unstable
                 sudo apt-get install -y postgis
                 sudo apt-get install postgresql

--- a/org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Nisida/ImportNisida.java
+++ b/org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Nisida/ImportNisida.java
@@ -472,7 +472,7 @@ public class ImportNisida {
 			}
 
 			/**
-			 * Create a FixWrapper on the parent track for the “Own Lat/Long position.
+			 * Create a FixWrapper on the parent track for the "Own Lat/Long position.
 			 * Course/speed are zeroes
 			 */
 

--- a/org.mwc.debrief.targetplatforms/eclipse-latest.target
+++ b/org.mwc.debrief.targetplatforms/eclipse-latest.target
@@ -67,17 +67,19 @@
 		  <unit id="org.eclipse.ui.views.log" version="1.3.100.v20210927-1100"/>
 	  </location>
     <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
-      <unit id="org.eclipse.ease.feature.feature.group" version="0.8.0.I202105061131"/>
-      <unit id="org.eclipse.ease.lang.javascript.feature.feature.group" version="0.8.0.I202104091054"/>
-      <unit id="org.eclipse.ease.lang.jvm.feature.feature.group" version="0.8.0.I202103261223"/>
-      <unit id="org.eclipse.ease.modules.charting.feature.feature.group" version="0.8.0.I202105070930"/>
-      <unit id="org.eclipse.ease.modules.feature.feature.group" version="0.8.0.I202105111149"/>
-      <unit id="org.eclipse.ease.modules.modeling.feature.feature.group" version="0.8.0.I202104061631"/>
-      <unit id="org.eclipse.ease.ui.feature.feature.group" version="0.8.0.I202105111204"/>
       <repository location="https://download.eclipse.org/ease/release/0.8.0"/>
+	    <unit id="org.eclipse.ease.feature.feature.group" version="0.8.0.I202105061131"/>
+	    <unit id="org.eclipse.ease.lang.javascript.feature.feature.group" version="0.8.0.I202104091054"/>
+	    <unit id="org.eclipse.ease.lang.jvm.feature.feature.group" version="0.8.0.I202103261223"/>
+	    <unit id="org.eclipse.ease.modules.charting.feature.feature.group" version="0.8.0.I202105070930"/>
+	    <unit id="org.eclipse.ease.modules.feature.feature.group" version="0.8.0.I202105111149"/>
+	    <unit id="org.eclipse.ease.modules.modeling.feature.feature.group" version="0.8.0.I202104061631"/>
+	    <unit id="org.eclipse.ease.ui.feature.feature.group" version="0.8.0.I202105111204"/>
 	    <unit id="org.eclipse.ease.lang.javascript.feature.source.feature.group" version="0.8.0.I202104091054"/>
 	    <unit id="org.eclipse.ease.lang.jvm.feature.source.feature.group" version="0.8.0.I202103261223"/>
+	    <unit id="org.eclipse.ease.lang.scriptarchive.feature.feature.group" version="0.8.0.I202104061519"/>
 	    <unit id="org.eclipse.ease.lang.scriptarchive.feature.source.feature.group" version="0.8.0.I202104061519"/>
+	    <unit id="org.eclipse.ease.lang.unittest.feature.feature.group" version="0.8.0.I202105030811"/>
 	    <unit id="org.eclipse.ease.lang.unittest.feature.source.feature.group" version="0.8.0.I202105030811"/>
 	    <unit id="org.eclipse.ease.ui.feature.source.feature.group" version="0.8.0.I202105111204"/>
     </location>

--- a/org.mwc.debrief.targetplatforms/eclipse-latest.target
+++ b/org.mwc.debrief.targetplatforms/eclipse-latest.target
@@ -34,7 +34,7 @@
       <unit id="org.eclipse.jpt.jpadiagrameditor.doc.user" version="1.2.102.v201902121834"/>
       <unit id="org.eclipse.jpt.jpadiagrameditor.ui" version="1.2.201.v201902121834"/>
       <unit id="org.eclipse.wtp.jee.capabilities" version="1.0.100.v201903222024"/>
-      <repository location="http://download.eclipse.org/webtools/repository/2021-03"/>
+      <repository location="https://download.eclipse.org/webtools/repository/2021-03"/>
     </location>
       <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
 	      <unit id="org.eclipse.wst.xml_ui.feature.feature.group" version="3.21.0.v202102222242"/>

--- a/org.mwc.debrief.targetplatforms/eclipse-latest.target
+++ b/org.mwc.debrief.targetplatforms/eclipse-latest.target
@@ -97,7 +97,7 @@
 		  <unit id="org.eclipse.nebula.visualization.feature.feature.group" version="2.1.0.202106201019"/>
 	  </location>
 	  <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
-		  <repository location="https://download.eclipse.org/webtools/repository/2021-09/"/>
+		  <repository location="https://download.eclipse.org/webtools/downloads/drops/R3.23.0/R-3.23.0-20210822084517/repository"/>
 		  <unit id="org.eclipse.jpt.assembly.feature.feature.group" version="2.3.3.v202007091849"/>
 		  <unit id="org.eclipse.jpt.common.eclipselink.feature.feature.group" version="1.3.200.v201903221940"/>
 		  <unit id="org.eclipse.jpt.common.feature.feature.group" version="1.5.100.v201903221940"/>

--- a/org.mwc.debrief.targetplatforms/eclipse-latest.target
+++ b/org.mwc.debrief.targetplatforms/eclipse-latest.target
@@ -97,7 +97,7 @@
 		  <unit id="org.eclipse.nebula.visualization.feature.feature.group" version="2.1.0.202106201019"/>
 	  </location>
 	  <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
-		  <repository location="http://download.eclipse.org/webtools/repository/2021-09/"/>
+		  <repository location="https://download.eclipse.org/webtools/repository/2021-09/"/>
 		  <unit id="org.eclipse.jpt.assembly.feature.feature.group" version="2.3.3.v202007091849"/>
 		  <unit id="org.eclipse.jpt.common.eclipselink.feature.feature.group" version="1.3.200.v201903221940"/>
 		  <unit id="org.eclipse.jpt.common.feature.feature.group" version="1.5.100.v201903221940"/>

--- a/org.mwc.debrief.targetplatforms/eclipse-latest.target
+++ b/org.mwc.debrief.targetplatforms/eclipse-latest.target
@@ -74,7 +74,7 @@
       <unit id="org.eclipse.ease.modules.feature.feature.group" version="0.8.0.I202105111149"/>
       <unit id="org.eclipse.ease.modules.modeling.feature.feature.group" version="0.8.0.I202104061631"/>
       <unit id="org.eclipse.ease.ui.feature.feature.group" version="0.8.0.I202105111204"/>
-      <repository location="https://download.eclipse.org/ease/release/latest"/>
+      <repository location="https://download.eclipse.org/ease/release/0.8.0"/>
 	    <unit id="org.eclipse.ease.lang.javascript.feature.source.feature.group" version="0.8.0.I202104091054"/>
 	    <unit id="org.eclipse.ease.lang.jvm.feature.source.feature.group" version="0.8.0.I202103261223"/>
 	    <unit id="org.eclipse.ease.lang.scriptarchive.feature.source.feature.group" version="0.8.0.I202104061519"/>

